commit d11d5b5dc75ede4b99947de743ae51bcb1d13c1a
Author: popcornmix <popcornmix@gmail.com>
Date:   Tue Feb 14 12:59:00 2017 +0000

    Revert "clk-bcm2835: Mark used PLLs and dividers CRITICAL"
    
    This reverts commit 9937447fd23d7ed191da8c505b8b34006d15b17e.

diff --git a/drivers/clk/bcm/clk-bcm2835.c b/drivers/clk/bcm/clk-bcm2835.c
index 0f0397e351e3..21e2a538ff0d 100644
--- a/drivers/clk/bcm/clk-bcm2835.c
+++ b/drivers/clk/bcm/clk-bcm2835.c
@@ -1214,11 +1214,6 @@ bcm2835_register_pll_divider(struct bcm2835_cprman *cprman,
 	divider->div.hw.init = &init;
 	divider->div.table = NULL;
 
-	if (!(cprman_read(cprman, data->cm_reg) & data->hold_mask)) {
-		init.flags |= CLK_IS_CRITICAL;
-		divider->div.flags |= CLK_IS_CRITICAL;
-	}
-
 	divider->cprman = cprman;
 	divider->data = data;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1156_d11d5b5dc75ede4b99947de743ae51bcb1d13c1a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1156_d11d5b5dc75ede4b99947de743ae51bcb1d13c1a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
